test-runtime/interval.test.ts

```diff
@@ -115,7 +115,7 @@ describe('interval selections at runtime in unit views', function() {
 
     embed(spec('unit', 1, {type, encodings: ['x']}, {values, x: {type: 'temporal', timeUnit: 'day'}}));
 
-    extents = [[1136188800000, 1136275200000, 1136361600000], [1136448000000, 1136534400000]];
+    extents = [[1136188800000, 1136361600000], [1136448000000, 1136534400000]];
```

Comment:
@arvind -- I have to make change to this line to make it work -- but not sure if it is really correct.  Could you please help double check?

Reply:
LGTM.